xml/System.Diagnostics/ActivityTraceId.xml

```diff
@@ -214,9 +214,9 @@ The characters in <paramref name="idData" /> are not all lower-case hexadecimal
         <Parameter Name="obj" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="obj">The <see cref="T:System.Object" /> to compare to.</param>
+        <summary>Determines whether two <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> objects have the same value.</summary>
+        <returns><see langword="true" /> if <paramref name="obj" /> is an instance of <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> and has the same hex value as <see langword="this" />; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="obj" /> is an instance of <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> and has the same hex value as the current instance; otherwise, <see langword="false" />.</returns>
```